### PR TITLE
Promo #47 - Robotic Pollinators

### PR DESCRIPTION
**Pull request by d-little**
### Robotic Pollinators

... not a particularly good card, but cute lil robees

image

Source: https://i.redd.it/70vv0axhf2la1.jpg

### CODE REVIEW

File: src/server/cards/promo/RobotPollinators.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+export class RobotPollinators extends Card implements IProjectCard {
+  constructor() {
+    super({
+      cardType: CardType.AUTOMATED,
+      name: CardName.ROBOT_POLLINATORS,
+      cost: 9,
+      requirements: CardRequirements.builder((b) => b.oxygen(4)),
+      behavior: {
+        production: {plants: 1},
+        stock: {titanium: 3},
```
Comment (kberg) [inline src/server/cards/promo/RobotPollinators.ts:21]: Gain 3 titanium?

You can instead do `stock: {plants: {tag: Tag.PLANT}` and remove bespokePlay.

**Review by kberg — APPROVED** [review]
One change. Well, two changes?

File: src/server/cards/promo/RobotPollinators.ts (hunk 1 above)
Comment (d-little) [reply src/server/cards/promo/RobotPollinators.ts]: Whoops!  I'll blame being distracted from real actual work

Comment (d-little) [reply src/server/cards/promo/RobotPollinators.ts]: .. I also really should start doing deeper sanity checks, for all production/resources, not just what I'm looking for.  Dont want to end up like Jurassic Park

**Pull request merged**